_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: heuristics
+title: Participatory design&#58; inclusivity heuristics
+permalink: /resources/participatory-design-inclusivity-heuristics/
+redirect_from:
+  - /participatory-design-inclusivity-heuristics/
+description: A set of heuristics for ensuring that a participatory design effort is proceeding in an inclusive way.
+---
+
+<h1>Participatory Design: Inclusivity Heuristics</h1>
+
+<p>The following indicators can be used to help determine if a <a href="{{site.baseurl}}/fundamentals/participatory-design">participatory design</a> process is proceeding in a way that encourages input from stakeholders and users. Indicators are spread across three broad categories: <a href="#account-for-context">Account for context</a>, <a href="#encourage-participation">Encourage participation</a>, and <a href="#inform-and-share-understanding">Inform and share understanding</a>.</p>
+
+<table>
+  <thead>
+    <tr>
+      <th scope="col" class="col-indicator">Indicator</th>
+      <th scope="col" class="col-why-it-matters">Why it matters</th>
+      <th scope="col" class="col-good-signs">Good signs</th>
+      <th scope="col" class="col-warning-signs">Warning signs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="4" id="account-for-context"><p>Account for context</p></th>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Values</p></th>
+      <td class="col-why-it-matters"><p>Values tie people together in ways that are difficult to break, especially when they’re not made explicit. Accounting for values helps teams approach design honestly and respectfully.</p></td>
+      <td class="col-good-signs"><p>The team identifies and reflects on its values. The team respects the values of the people affected by its work. The team acknowledges when it seeks to challenge the values of others.</p></td>
+      <td class="col-warning-signs"><p>The team doesn’t identify or discuss its values. The team does not see or consider the merits of the values of others. The team misleads others when attempting to change their values.</p></td>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Power dynamics</p></th>
+      <td class="col-why-it-matters"><p>Power dynamics shape all interpersonal relationships, as well as our collective ability to imagine potential futures. Power dynamics affect what’s considered possible, and the theories of change we will knowingly pursue.</p></td>
```

Comment:
This paragraph is well-written, but requires a lot of background knowledge to interpret. Can you give an example of imagining potential futures during a design process? How would you explain what you mean to someone who doesn't know the term "theories of change"?

Reply:
And what does "power" mean in this context? Organizational seniority? Authority? Social status? (I mean, probably all of the above.) "Power" is an abstract concept. How might you bring what you mean closer to the experience of likely readers?

Reply:
>Can you give an example of imagining potential futures during a design process?
Yes! This is the role of design fiction, and sometimes prototyping. When I detailed to the 18F Practice team I wrote a series of ["vignettes"](https://docs.google.com/document/d/1qQrvz6KXFJu9xm3_yA3FPIseRRvqEoVf3K-fQpbd3Vs/edit) and [draft press releases](https://docs.google.com/document/d/1L6Lsf6HAS7KrRTZln4qULIbT1cTg5-7N298jGeCuk80/edit#heading=h.o2r4noogl5n) to describe what success might look like.

Reply:
>"Power" is an abstract concept. How might you bring what you mean closer to the experience of likely readers?

I agree, and I'm not sure how I can do this. I want this to be written so that even our partners could point to this to assess how well they're being included (for example, within their own org) or including people outside of their 18F engagement.